src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -211,6 +213,11 @@ public override void OnStopActivity(Activity activity, object payload)
                 // the one created by the instrumentation.
                 // And retrieve it here, and set it to Current.
             }
+
```

Comment:
The if block above this would never be true if we reuse the operation name right?
`if (activity.OperationName.Equals(ActivityNameByHttpInListener, StringComparison.Ordinal))
            {
`

Reply:
Yeah, if we decide to go forward with this approach, we can get rid of the above if block.